lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -420,15 +420,35 @@ class MongoClientInterface {
              * code is the one related to mentionned issue.
              */
             if (err) {
-                if (!err.index || err.index !== 1
-                 || !err.code || err.code !== 11000) {
-                    log.error(
-                        'putObjectVerCase1: error putting object version',
-                    { error: err.errmsg });
+                if (err.code === 11000) {
+                    log.debug('putObjectVerCase1: error putting object version', {
+                        code: err.code,
+                        error: err.errmsg,
+                        isRetry: isRetry ? 'true' : 'false',
```

Comment:
bools have `.toString()` methods if you want to replace this ternary
```suggestion
                        isRetry: isRetry.toString(),
```

Reply:
i'm keeping it that way since isRetry may be undefined